lib/Mojolicious/Static.pm

```diff
@@ -62,7 +62,9 @@ sub is_fresh {
   my $res_headers = $c->res->headers;
   my ($last, $etag) = @$options{qw(last_modified etag)};
   $res_headers->last_modified(Mojo::Date->new($last)->to_string) if $last;
-  $res_headers->etag($etag = qq{"$etag"}) if $etag;
+  $res_headers->etag($etag
+      = (substr($etag, 0, 3) eq qq{W/"} ? $etag : qq{"$etag"}))
```

Comment:
Why not something like `$etag =~ m!^W/"!`?

Reply:
Is `substr` not more efficient? I don't care either way.

Reply:
Is that even measurable here? Regex would make this one easy to read line of code.